idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -670,18 +692,32 @@ def uploader_check_wait_all():
             assert t.completed and not t.exception
     else:
         # run STAR
+        total_counts_from_star = {} # getting total_reads and file truncation status from STAR
         logparams = return_merged_dict(
             DEFAULT_LOGPARAMS,
             {"title": "STAR",
              "version_file_s3": STAR_BOWTIE_VERSION_FILE_S3,
              "output_version_file": os.path.join(RESULT_DIR, VERSION_OUT)})
-        run_and_log_s3(logparams, target_outputs["run_star"], lazy_run, SAMPLE_S3_OUTPUT_PATH, run_star, fastq_files, uploader_start)
+        run_and_log_s3(logparams, target_outputs["run_star"], lazy_run, SAMPLE_S3_OUTPUT_PATH,
+                       run_star, fastq_files, uploader_start, total_counts_from_star)
+        if not total_counts_from_star.get('total_reads'):
+            # Total reads not set. Most likely it's lazy run. Damn it. would have to actually count the reads.
+            # TODO: Remove this one we also lazy load the stats.json file
+            total_counts_from_star['total_reads'] = count_reads(fastq_files[0], initial_file_type_for_log)
+
+        stats.data.append(total_counts_from_star)
```

Comment:
Does it matter that total_counts_from_star may have a "truncated" key?   I am guessing not.